Mod Menu Helper/assets/modmenu/lang/tr_TR.json

```diff
@@ -0,0 +1,55 @@
+{
```

Comment:
Please rename the file to `tr_tr.json`. Minecraft uses lowercase language codes for a long time, though I can't remember why they actually changed it.